### PR TITLE
DEV: Reduce jQuery usage in acceptance tests

### PR DESCRIPTION
**Pull request by CvX**

### CODE REVIEW

File: app/assets/javascripts/discourse/tests/acceptance/category-new-test.js [hunk 1]
```diff
@@ -30,21 +30,21 @@ acceptance("Category New", function (needs) {
...
+      query(".edit-category-title h2").innerText,
       I18n.t("category.edit_dialog_title", {
         categoryName: "testing",
       })
     );
 
     await click(".edit-category-security a");
     assert.ok(
-      queryAll("button.edit-permission"),
+      exists(".permission-row button.reply-toggle"),
```
Comment (hnb-ku) [inline app/assets/javascripts/discourse/tests/acceptance/category-new-test.js:41]: I'm guessing this is just a better selector, yeah?

**Review by hnb-ku — APPROVED** [review]

**Comment by hnb-ku** [comment]
This is a very cool PR :pray: Only one minor note but otherwise, this looks great!

File: app/assets/javascripts/discourse/tests/acceptance/category-new-test.js (hunk 1 above)
Comment (CvX) [reply app/assets/javascripts/discourse/tests/acceptance/category-new-test.js:41]: Yeah, better - this one actually exists 🤣

`queryAll()` always returns a (truthy) jQ object, so doing `assert.ok(queryAll())` does not check anything at all 😄

Comment (hnb-ku) [reply app/assets/javascripts/discourse/tests/acceptance/category-new-test.js:41]: Thanks for the details!

*shakes fists at jQuery*

I don't have any more concerns :+1:

**Pull request merged**